src/platform-aws.cpp

```diff
@@ -133,6 +133,17 @@ static struct ec2_platform_data platform_data_map[] = {
 			{ "NCCL_NET_FORCE_FLUSH", "0" },
 		},
 	},
+	{
+		.name = "p5.4xlarge",
+		.regex = NULL,
+		.topology = NULL,
+		.default_dup_conns = 0,
+		.latency = 75.0,
+		.gdr_required = false,
+		.default_protocol = PROTOCOL::SENDRECV,
+		.domain_per_thread = false,
+		.env = {},
+	},
 	{
```

Comment:
Will this conflict with L149's wildcard? Or this will be preferred as the first match of `p5.x4xlarge`